src/app/app.module.ts

```diff
@@ -68,17 +62,23 @@ const matomoTrackers = environment.matomoSiteId ? [
     DatePipe,
     {
       provide: APP_BASE_HREF,
-      useValue: environment.donateGlobalUriPrefix,
+      useValue: environment.donateUriPrefix,
     },
     { provide: TBG_DONATE_ID_STORAGE, useExisting: LOCAL_STORAGE },
     { provide: TBG_DONATE_STORAGE, useExisting: LOCAL_STORAGE },
     { provide: MAT_CHECKBOX_DEFAULT_OPTIONS, useValue: { color: 'primary' } },
     { provide: MAT_RADIO_DEFAULT_OPTIONS, useValue: { color: 'primary' } },
-    { provide: RECAPTCHA_NONCE, useValue: environment.recaptchaNonce },
     {
-      provide: RECAPTCHA_BASE_URL,
-      useValue: 'https://recaptcha.net/recaptcha/api.js'  // using this URL instead of default google.com means we avoid google.com cookies.
-    }
+      provide: RECAPTCHA_LOADER_OPTIONS,
+      useValue: {
+        onBeforeLoad(_url: any) {
```

Comment:
From looking at https://www.npmjs.com/package/ng-recaptcha I think ng-recaptcha will pass a URL object here. Even if we don't use it I'd prefer to have the real type (or even `unknown` ) rather than `any`. 
```suggestion
        onBeforeLoad(_url: URL) {
```